rel-eng/tito.props

```diff
@@ -525,3 +526,8 @@ whitelist = katello
   rubygem-runcible
   rubygem-typhoeus
   rubygem-zest
+
+
+[rhel7]
```

Comment:
This looks odd

Reply:
This is `./add_gem_package.sh rubygem-fog-proxmox scl foreman-plugins-nightly-rhel7` script I launched which did it. I think it sorts list in alphabetic order. I did not edit manually this file.

Reply:
Perhaps you called it with a space and then rhel7? Normally this section isn't supposed to exist.

Reply:
I can't remember. Ok I will start again.

Reply:
Perhaps it's easier to do `git revert 2cc510d33c02e9ec42e005a2cc38e944ef378506 --no-commit && ./bump_rpm.sh packages/plugin/rubygem-fog-proxmox`

Reply:
I have tried in a new fresh branch. I have launched `./add_gem_package.sh fog-proxmox scl foreman-plugins-nightly-rhel7` with no space in one command line. And the script did the same commits: rearrange foreman-task. Have you any idea what command I should launch instead?

Reply:
git revert solution is not so easier: many changes to check... The issue you checked comes from `./add_gem_package.sh` script which sorts foreman-tasks. It does not break the build.

Reply:
Sorting tito.props whitelist is here in the code: https://github.com/theforeman/foreman-packaging/blob/401a1fe288b56283be0d35a2476ad449a7f76baa/add_gem_package.sh#L71

Reply:
I have still done the changes you asked for.